### PR TITLE
Change the sort order for the organisation usage page

### PR DESCRIPTION
**Pull request by servingUpAces**
Ensure the archived services are at the bottom of the list. The organisation trial mode page already sorts the archived services to the bottom.

### CODE REVIEW

File: app/organisation/rest.py [hunk 1]
```diff
@@ -135,7 +135,7 @@ def get_organisation_services_usage(organisation_id):
         return jsonify(result='error', message='No valid year provided'), 400
     services = fetch_usage_year_for_organisation(organisation_id, year)
     list_services = services.values()
-    sorted_services = sorted(list_services, key=lambda s: s['service_name'].lower())
+    sorted_services = sorted(list_services, key=lambda s: (-s['active'], s['service_name'].lower()))
```
Comment (CrystalPea) [inline app/organisation/rest.py:138]: why is the dash there? Sorry if that's a silly question 😅

Comment (quis) [reply app/organisation/rest.py:138]:
```python
-True
>>> -1

-False
>>> 0
```
😨 I have learned something new about Python today

Comment (servingUpAces) [reply app/organisation/rest.py:138]: The dash reverses the order, so I get True before False

Comment (CrystalPea) [reply app/organisation/rest.py:138]: sweet, thanks a lot, I had no idea this is a thing, today I've learnt :3

**Review by CrystalPea — APPROVED** [review]

**Pull request merged**